docs/101/allocations.md

```diff
@@ -26,23 +26,73 @@ accounts balance
 ```
 
 ## How do I review the usage of my allocation?
-The `accounts` tool provides several options for summarizing allocation usage. To obtain a summary, execute the command:
+The `accounts` tool provides several options for summarizing the usage of the allocations to which your RCC account is accessible. To obtain a summary, execute the command `accounts usage` with the account name(s):
+
+
+=== "Midway2, DaLI"
+      ```
+      accounts usage
+      accounts usage --account [pi-cnetid]
+      accounts usage --accounts [pi-cnetid1],[pi-cnetid2]
+      ```
+      
+===+ "Midway3, Beagle3"
+      ```
+      accounts usage
+      accounts usage -a [pi-cnetid]
+      ```
+
+where `[pi-cnetid]` is the name of the account, often your PI's.
+
+You may use the `--byuser` option to view individual usage by group members:
+
+=== "Midway2, DaLI"
+      ```
+      accounts usage
+      accounts usage --account [pi-cnetid] --byuser
+      ```
+
+===+ "Midway3, Beagle3"
+      ```
+      accounts usage
+      accounts usage -a [pi-cnetid] -byuser
+      ```
+
+You can also use the `rcchelp` command (a wrapper of system-provided tools including `accounts`), which requires slightly different arguments (note only one character '-')
+
+=== "Midway2, DaLI"
+      ```
+      rcchelp usage 
+      rcchelp usage -account [pi-cnetid] -byuser
+      ```
+
+=== "Midway3, Beagle3"
+      ```
+      rcchelp usage 
+      rcchelp usage -account [pi-cnetid] -byuser
+      ```
 
-```
-accounts usage
-```
+---
 
-To view usage per job, utilize the following command:
+To view the resource usage per job, you need to run on Midway2 before using either `accounts`:
 
-```
-accounts usage --byjob
-```
+=== "Midway2, DaLI"
+      ```
+      accounts usage --byjob
+      accounts usage --account [pi-cnetid] --byjob
+      ```
```

Comment:
works fine.

something related: help show single dash for "byjob", but it works fine with both single/double. same for "account" option.